Docs: Fixed Grafana Fundamentals tutorial to match updated UI

glitchwizard opened the pull request:
<!--

Thank you for sending a pull request! Here are some tips:

1. If this is your first time, please read our contribution guide at https://github.com/grafana/grafana/blob/main/CONTRIBUTING.md

2. Ensure you include and run the appropriate tests as part of your Pull Request.

3. In a new feature or configuration option, an update to the documentation is necessary. Everything related to the documentation is under the docs folder in the root of the repository.

4. If the Pull Request is a work in progress, make use of GitHub's "Draft PR" feature and mark it as such.

5. If you can not merge your Pull Request due to a merge conflict, Rebase it. This gets it in sync with the main branch.

6. Name your PR as "<FeatureArea>: Describe your change", e.g. Alerting: Prevent race condition. If it's a fix or feature relevant for the changelog describe the user impact in the title. The PR title is used to auto-generate the changelog for issues marked with the "add to changelog" label.

7. If your PR content should be added to the What's New document for the next major or minor release, add the **add to what's new** label to your PR. Note that you should add this label to the main PR that introduces the feature; do not add this label to smaller PRs for the feature.

-->

**What is this feature?**

It's updating the Grafana fundamentals tutorial docs. The word `Configuration` was used to describe an action for getting started, when the actual UI has the word `Administration` and not `Configuration`. Updated the docs to reflect the actual environment created.

**Why do we need this feature?**

It makes the docs most accurate.

**Who is this feature for?**

Everyone

**Which issue(s) does this PR fix?**:

<!--

- Automatically closes linked issue when the Pull Request is merged.

Usage: "Fixes #<issue number>", or "Fixes (paste link of issue)"

-->

Fixes # N/A

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

glitchwizard committed:
Update index.md

The word `Configuration` was used to describe an action for getting started, when the actual UI has the word `Administration` and not Configuration. Updated the docs to reflect the actual environment created.

glitchwizard committed:
Merge pull request #1 from glitchwizard/glitchwizard-fundamentals-docs-update

Grafana Fundamentals Docs: Update incorrect documentation

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=73168) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=73168) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/grafana?pullRequest=73168) it.</sub>

imatwawana reviewed (commented):
Thanks for the heads up! It looks like the navigation has changed in a few spots, so I'd like to fix them all at once. Is it okay with you if I pull in your branch to make additional changes?

glitchwizard commented:
@imatwawana go for it. I did this before I finished the article, I was going to do the same because yeah, like you said, I noticed a few spots where it could be updated. If you want, I can go through and do it too, either way.

glitchwizard commented:
Ok sounds good. I'll get on it right now.

On Mon, Aug 14, 2023 at 1:57 PM Isabel ***@***.***> wrote:

> @imatwawana <https://github.com/imatwawana> go for it. I did this before
> I finished the article, I was going to do the same because yeah, like you
> said, I noticed a few spots where it could be updated. If you want, I can
> go through and do it too, either way.
>
> I'm happy to leave it with you so the full PR glory can be yours! Just
> re-request review from me when you're ready for me to take another look.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/grafana/grafana/pull/73168#issuecomment-1678050304>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJVGMNE52SCVYB3K4WCDDT3XVKGNBANCNFSM6AAAAAA3L5GNYQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Charley McGowan*
~~~   ~~~~   ~~~

glitchwizard committed:
Update index.md

Updated the tutorial to match the current UI options

glitchwizard committed:
Merge branch 'grafana:main' into main

glitchwizard committed:
Merge pull request #2 from glitchwizard/glitchwizard-fundamentals-docs-update

Further updates for the tutorial documentation to match the most current UI options (buttons and the like)

glitchwizard commented:
ok @imatwawana the latest PR on my local branch should have all the updates. In some cases I had to rewrite some sentences because the actual user flow was just a bit different now in the UI, than what the tutorial actually had. 

I hope it's concise and clear. Please let me know if there should be any changes made.

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -116,19 +116,19 @@ Grafana is an open-source platform for monitoring and observability that lets yo
 To be able to visualize the metrics from Prometheus, you first need to add it as a data source in Grafana.
 
-1. In the sidebar, hover over the **Configuration** (gear) icon, and then click **Data sources**.
+1. In the sidebar, hover over the **Administration** (gear) icon, and then click **Data sources**.
```
```suggestion
1. In the sidebar, click **Connections** and then **Data sources**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -116,19 +116,19 @@ Grafana is an open-source platform for monitoring and observability that lets yo
 1. In the list of data sources, click **Prometheus**.
 1. In the URL box, enter **http\://prometheus:9090**.
-1. Click **Save & test**.
+1. Scroll to the bottom and click **Save & test**.
```
```suggestion
1. Scroll to the bottom of the page and click **Save & test**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -165,19 +166,19 @@ Grafana Explore is a workflow for troubleshooting and data exploration. In this
 
-1. In the sidebar, hover over the **Configuration** (gear) icon, and then click **Data Sources**.
-1. Click **Add data source**.
+1. In the sidebar, hover over the **Administration** (gear) icon, and then click **Data Sources**.
```
```suggestion
1. In the sidebar, click **Connections** and then **Data sources**.
```

glitchwizard replied:
So just out of curiosity, what's the difference between a data source and a connection? And is there anything different between setting it up using the Admin way, or the Connections way?

imatwawana replied:
> So just out of curiosity, what's the difference between a data source and a connection? And is there anything different between setting it up using the Admin way, or the Connections way?

A data source is just a type of connection. In the Cloud version of Grafana, this functionality has been removed from Admin and only exists under Connections. I suspect this will happen in the on-prem version as well, so future-proofing here.

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -222,15 +224,17 @@ Dashboards consist of _panels_, each representing a part of the story you want y
 
-1. In the sidebar, hover over the **Create** (plus sign) icon and then click **Dashboard**.
-1. Click **Add new panel**. The Prometheus data source that you added will already be selected.
+1. Open the sidebar, then click **Dashboards**.
```
```suggestion
1. Open the sidebar and then click **Dashboards**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -222,15 +224,17 @@ Dashboards consist of _panels_, each representing a part of the story you want y
 
-1. In the sidebar, hover over the **Create** (plus sign) icon and then click **Dashboard**.
-1. Click **Add new panel**. The Prometheus data source that you added will already be selected.
+1. Open the sidebar, then click **Dashboards**.
+1. Click the **New** button on the right hand side.
+1. From the dropdown that appears, click **New Dashboard**
+1. This opens up an empty dashboard, click the **+ Add visualization** button
+1. Select Prometheus as the data source.
```
```suggestion
1. In the sidebar, click **Dashboards**.
1. On the **Dashboards** page, click **New** in top right corner and select **New Dashboard** in the drop-down.
1. Click **+ Add visualization**.
1. In the modal that opens, select the Prometheus data source that you just added.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
````diff
@@ -222,15 +224,17 @@ Dashboards consist of _panels_, each representing a part of the story you want y
    ```
 
-1. In the **Legend** field, enter **{{route}}** to rename the time series in the legend. The graph legend updates when you click outside the field.
 1. In the panel editor on the right, under **Panel options**, change the panel title to "Traffic".
````
```suggestion
1. Click **Options** under the query field to open the panel.
1. In the **Legend** field, select Custom and then enter `route` to rename the series in the legend. The graph legend updates when you click outside the field.
1. In the panel editor on the right, under **Panel options**, change the panel title to "Traffic".
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -270,9 +274,9 @@ Manually annotating your dashboard is fine for those single events. For regularl
+1. Click **Apply**. Grafana displays the Annotations list, with your new annotation.
 1. Click the **Go back** arrow to return to your dashboard.
-1. At the top of your dashboard, there is now a toggle to display the results of the newly created annotation query. Press it so that it's enabled.
+1. At the top of your dashboard, there is now a toggle with the name **Errors** to display the results of the newly created annotation query. Press it so that it's enabled.
```
```suggestion
1. At the top of your dashboard, there is now a toggle with the name **Errors** to display the results of the newly created annotation query. 
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -306,10 +310,10 @@ Your request bin is now waiting for the first request.
 
-1. Return to Grafana. In Grafana's sidebar, hover over the **Alerting** (bell) icon and then click **Contact points**.
-1. Click **+ New contact point**.
+1. Return to Grafana. In Grafana's sidebar, expand the **Alerting** (bell) list and then click **Contact points**.
```
```suggestion
1. Return to Grafana. In Grafana's sidebar, click **Alerting** and then **Contact points**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
-1. In Grafana's sidebar, hover over the **Alerting** (bell) icon and then click **Alert rules**.
-1. Click **+ New alert rule**.
-1. For **Section 1**, name the rule `fundamentals-test`, and set **Rule type** to **Grafana Managed Alert**. For **Folder** type `fundamentals` and in the box that appears, press **Create: fundamentals**.
+1. In Grafana's sidebar, expand the **Alerting** list (bell icon) and then click **Alert rules**.
```
```suggestion
1. In Grafana's sidebar, click **Alerting** and then **Alert rules**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
-1. For **Section 1**, name the rule `fundamentals-test`, and set **Rule type** to **Grafana Managed Alert**. For **Folder** type `fundamentals` and in the box that appears, press **Create: fundamentals**.
+1. In Grafana's sidebar, expand the **Alerting** list (bell icon) and then click **Alert rules**.
+1. Click the **Create alert rule** button.
+1. For **Section 1**, name the rule `fundamentals-test`, and set **Rule type** to **Grafana Managed Alert**.
```
```suggestion
1. For **Section 1**, name the rule `fundamentals-test`.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
+1. In Grafana's sidebar, expand the **Alerting** list (bell icon) and then click **Alert rules**.
+1. Click the **Create alert rule** button.
+1. For **Section 1**, name the rule `fundamentals-test`, and set **Rule type** to **Grafana Managed Alert**.
 1. For **Section 2**, find the **query A** box. Choose your Prometheus datasource and enter the same query that we used in our earlier panel: `sum(rate(tns_request_duration_seconds_count[5m])) by(route)`. Press **Run queries**. You should see some data in the graph.
```
```suggestion
1. For **Section 2**, set **Rule type** to **Grafana Managed Alert**. Next, find the **query A** box. Choose your Prometheus datasource and enter the same query that we used in our earlier panel: `sum(rate(tns_request_duration_seconds_count[5m])) by(route)`. Press **Run queries**. You should see some data in the graph.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
 1. For **Section 2**, find the **query A** box. Choose your Prometheus datasource and enter the same query that we used in our earlier panel: `sum(rate(tns_request_duration_seconds_count[5m])) by(route)`. Press **Run queries**. You should see some data in the graph.
-1. Now scroll down to the **query B** box. For **Operation** choose `Classic condition`. [You can read more about classic and multi-dimensional conditions here](/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`
-1. In **Section 3**, enter `30s` for the **Evaluate every** field. For the purposes of this tutorial, the evaluation interval is intentionally short. This makes it easier to test. In the **for** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.
+1. Now scroll down to the **query B** box. For **Operation** choose `Classic condition`. [You can read more about classic and multi-dimensional conditions here](/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`. **query C** should be deleted to make this ...
```
```suggestion
1. Now scroll down to the query box **B**. For the operation, change it from **Reduce** to **Classic condition**. [You can read more about classic and multi-dimensional conditions here](/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`. Delete query **C**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
-1. Now scroll down to the **query B** box. For **Operation** choose `Classic condition`. [You can read more about classic and multi-dimensional conditions here](/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`
-1. In **Section 3**, enter `30s` for the **Evaluate every** field. For the purposes of this tutorial, the evaluation interval is intentionally short. This makes it easier to test. In the **for** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.
+1. Now scroll down to the **query B** box. For **Operation** choose `Classic condition`. [You can read more about classic and multi-dimensional conditions here](/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`. **query C** should be deleted to make this ...
+1. In **Section 3**, Select **+ Add New** under **Folder** dropdown menu. Name the new folder `fundamentals` and press enter. This will create the folder needed. In **Evaluation Group** name it also `fundamentals` for now. In the **for** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.
```
```suggestion
1. In section **3**, Select **+ Add New** in **Folder** dropdown menu. Name the new folder `fundamentals` and press enter. This will create the folder needed. In **Evaluation Group**, name the group also `fundamentals` for now. In the **for** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
-1. In **Section 3**, enter `30s` for the **Evaluate every** field. For the purposes of this tutorial, the evaluation interval is intentionally short. This makes it easier to test. In the **for** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.
+1. Now scroll down to the **query B** box. For **Operation** choose `Classic condition`. [You can read more about classic and multi-dimensional conditions here](/docs/grafana/latest/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule/#single-and-multi-dimensional-rule). For conditions enter the following: `WHEN last() OF A IS ABOVE 0.2`. **query C** should be deleted to make this ...
+1. In **Section 3**, Select **+ Add New** under **Folder** dropdown menu. Name the new folder `fundamentals` and press enter. This will create the folder needed. In **Evaluation Group** name it also `fundamentals` for now. In the **for** field, enter `0m`. This setting makes Grafana wait until an alert has fired for a given time before Grafana sends the notification.
 1. In **Section 4**, you can add some sample text to your summary message. [Read more about message templating here](/docs/grafana/latest/alerting/unified-alerting/message-templating/).
```
```suggestion
1. In section **4**, you can add some sample text to your summary message. [Read more about message templating here](/docs/grafana/latest/alerting/unified-alerting/message-templating/).
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
 1. In **Section 4**, you can add some sample text to your summary message. [Read more about message templating here](/docs/grafana/latest/alerting/unified-alerting/message-templating/).
-1. Click **Save and exit** at the top of the page.
-1. In Grafana's sidebar, hover over the **Alerting** (bell) icon and then click **Notification policies**.
+1. Click **Save rule and exit** at the top of the page.
```
```suggestion
1. In the top right corner of the page, click **Save rule and exit**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
-1. Click **Save and exit** at the top of the page.
-1. In Grafana's sidebar, hover over the **Alerting** (bell) icon and then click **Notification policies**.
+1. Click **Save rule and exit** at the top of the page.
+1. Expand Grafana's sidebar, expand the **Alerting** (bell icon) node and then click **Notification policies**.
```
```suggestion
1. In the sidebar, click **Notification policies**.
```

imatwawana replied:
> Shouldn't we make it more clear where the 'Notification Policies' is located? Assuming the **Alerting** node is expanded isn't quite so clear. Just in case someone opens the sidebar and doesn't have it expanded for some reason.

I was walking my way through the tutorial as I made edits, so at this particular point in the steps, the sidebar should still be open (not the case all the time during the tutorial). Also, at this point, we'll have users 3 or 4 times to click the menu icon to open the sidebar, so if they have managed to close it, they've learned that behaviour by now.

imatwawana committed:
Made copy edits for styling and consistency with Cloud fundamentals tutorial

Made some copy edits to match our style guide and to make this tutorial a bit more consistent with the Cloud version. Also added the step for updating the legend back to the build dashboards section.

imatwawana commented:
@glitchwizard - Thank you for all your work on this! I committed a number of style edits that are just to bring the tutorial into closer alignment with our overall style, which is a constant work in progress. I'm going to take one more look at this tomorrow to make sure I haven't missed anything and to resolve any failing checks. Then we should be good to merge this PR.

glitchwizard commented:
@imatwawana rock on, thank you. Let me know if there's anything else I should do.

imatwawana committed:
Ran linter

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -140,7 +140,8 @@ Grafana Explore is a workflow for troubleshooting and data exploration. In this
 
 1. In the sidebar, click the **Explore** (compass) icon. The Prometheus data source that you added will already be selected.
```
```suggestion
1. Click the menu icon and, in the sidebar, click **Explore**. The Prometheus data source that you added will already be selected.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -140,7 +140,8 @@ Grafana Explore is a workflow for troubleshooting and data exploration. In this
 1. In the sidebar, click the **Explore** (compass) icon. The Prometheus data source that you added will already be selected.
 1. In the **Query editor**, where it says _Enter a PromQL query…_, enter `tns_request_duration_seconds_count` and then press Shift + Enter.
-   A graph appears.
+   A graph appears. Note, it may start you in the The Prometheus query builder, if this is the case, select the `Code` toggle in the top right of the query window, just below the `Run Query` button.
```
```suggestion
   A graph appears. 
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -140,7 +140,8 @@ Grafana Explore is a workflow for troubleshooting and data exploration. In this
 
 1. In the sidebar, click the **Explore** (compass) icon. The Prometheus data source that you added will already be selected.
 1. In the **Query editor**, where it says _Enter a PromQL query…_, enter `tns_request_duration_seconds_count` and then press Shift + Enter.
```
```suggestion
1. Confirm that you're in code mode by checking the **Builder/Code** toggle at the top right corner of the query panel. 
1. In the **Query editor**, where it says _Enter a PromQL query…_, enter `tns_request_duration_seconds_count` and then press Shift + Enter.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -140,7 +140,8 @@ Grafana Explore is a workflow for troubleshooting and data exploration. In this
 1. In the sidebar, click the **Explore** (compass) icon. The Prometheus data source that you added will already be selected.
 1. In the **Query editor**, where it says _Enter a PromQL query…_, enter `tns_request_duration_seconds_count` and then press Shift + Enter.
-   A graph appears.
+   A graph appears. Note, it may start you in the The Prometheus query builder, if this is the case, select the `Code` toggle in the top right of the query window, just below the `Run Query` button.
+   <img width="167" alt="image" src="https://github.com/glitchwizard/grafana/assets/40527412/f5247425-ab0b-4e78-a608-feb11c3b4b8e">
```
```suggestion
   A graph appears. 
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -165,19 +166,19 @@ Grafana Explore is a workflow for troubleshooting and data exploration. In this
 
-1. In the sidebar, hover over the **Configuration** (gear) icon, and then click **Data Sources**.
-1. Click **Add data source**.
+1. In the sidebar, click **Connections** and then **Data sources**.
```
```suggestion
1. Click the menu icon and, in the sidebar, click **Connections** and then **Data sources**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -222,15 +223,18 @@ Dashboards consist of _panels_, each representing a part of the story you want y
 
-1. In the sidebar, hover over the **Create** (plus sign) icon and then click **Dashboard**.
-1. Click **Add new panel**. The Prometheus data source that you added will already be selected.
+1. In the sidebar, click **Dashboards**.
```
```suggestion
1. Click the menu icon and, in the sidebar, click **Dashboards**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -222,15 +223,18 @@ Dashboards consist of _panels_, each representing a part of the story you want y
 
-1. In the **Legend** field, enter **{{route}}** to rename the time series in the legend. The graph legend updates when you click outside the field.
+1. Click **Options** under the query field to open the panel.
+1. In the **Legend** field, select Custom and then enter `route` to rename the series in the legend. The graph legend updates when you click outside the field.
```
```suggestion
1. In the **Legend** field, select **Custom** and then enter `route` to rename the series in the legend. The graph legend updates when you click outside the field.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -270,9 +274,9 @@ Manually annotating your dashboard is fine for those single events. For regularl
 <!--this add button is gone rn. look into this -->
 
-1. Click **Add**. Grafana displays the Annotations list, with your new annotation.
+1. Click **Apply**. Grafana displays the Annotations list, with your new annotation.
```
```suggestion
1. Click **Apply**. Grafana displays the **Annotations** page, with your new annotation in the list.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -270,9 +274,9 @@ Manually annotating your dashboard is fine for those single events. For regularl
 
-1. Click **Add**. Grafana displays the Annotations list, with your new annotation.
+1. Click **Apply**. Grafana displays the Annotations list, with your new annotation.
 1. Click the **Go back** arrow to return to your dashboard.
```
```suggestion
1. Click the name of the dashboard in the breadcrumb at the top of the screen to return to your dashboard.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -270,9 +274,9 @@ Manually annotating your dashboard is fine for those single events. For regularl
+1. Click **Apply**. Grafana displays the Annotations list, with your new annotation.
 1. Click the **Go back** arrow to return to your dashboard.
-1. At the top of your dashboard, there is now a toggle to display the results of the newly created annotation query. Press it so that it's enabled.
+1. At the top of your dashboard, there is now a toggle with the name **Errors** to display the results of the newly created annotation query.
```
```suggestion
1. At the top of your dashboard, there is now a toggle with the name **Errors** to display the results of the newly created annotation query. The toggle is enabled by default.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -306,10 +310,10 @@ Your request bin is now waiting for the first request.
 
-1. Return to Grafana. In Grafana's sidebar, hover over the **Alerting** (bell) icon and then click **Contact points**.
-1. Click **+ New contact point**.
+1. Return to Grafana. In Grafana's sidebar, click **Alerting** and then **Contact points**.
```
```suggestion
1. Return to Grafana. Click the menu icon and, in the sidebar, click **Alerting** and then **Contact points**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
-1. In **Section 4**, you can add some sample text to your summary message. [Read more about message templating here](/docs/grafana/latest/alerting/unified-alerting/message-templating/).
-1. Click **Save and exit** at the top of the page.
-1. In Grafana's sidebar, hover over the **Alerting** (bell) icon and then click **Notification policies**.
+1. In Grafana's sidebar, click **Alerting** and then **Alert rules**.
```
```suggestion
1. In the sidebar, click **Alert rules**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
-1. Click **Save and exit** at the top of the page.
-1. In Grafana's sidebar, hover over the **Alerting** (bell) icon and then click **Notification policies**.
+1. In Grafana's sidebar, click **Alerting** and then **Alert rules**.
+1. Click the **Create alert rule** button.
```
```suggestion
1. Click **Create alert rule**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -321,15 +325,15 @@ We have now created a dummy webhook endpoint and created a new Alerting Contact
+1. In section **4**, you can add some sample text to your summary message. [Read more about message templating here](/docs/grafana/latest/alerting/unified-alerting/message-templating/).
+1. In the top right corner of the page, click **Save rule and exit**.
+1. In the sidebar, click **Notification policies**.
 1. Under **Default policy**, select **...** &rsaquo; **Edit** and change the **Default contact point** to **RequestBin**.
```
```suggestion
1. Under **Default policy**, click the button with the three dots and select **Edit**. Change the **Default contact point** to **RequestBin**. Click **Update default policy**.
```

imatwawana committed:
Edited text to match updated UI

Made edits to match UI behaviour as well as some further style edits.

imatwawana committed:
Made final copy edits
